### ISSUE TITLE
It asks me for a module pip cannot install

### ISSUE THREAD

**Issue by juangea** [sammyf/moztts#2]
I'm on Windows 10, and when I try to use it (after installing requirements) I get this error:

raise Exception(" [!] No espeak backend found. Install espeak-ng or espeak to your system.")
Exception:  [!] No espeak backend found. Install espeak-ng or espeak to your system.

If I try to use pip install and one of those modeuls pip says it cannot find any version.

Some ideas?

Thanks!

**Comment by sammyf** [comment]
Do you know which module can not be found? I only have Linux machines here, so it's a bit hard to debug, but the main component is really just https://github.com/mozilla/TTS . Can you try to install that?

**Comment by bigsk1** [comment]
the new oogabooga update calls for numpy=1.24 which doesn't work with TTS 
tts 0.16.3 requires numpy==1.22.0; python_version <= "3.10", but you have numpy 1.24

maybe related

**Comment by sammyf** [comment]
hmm .. I just updated numpy from 1.22.0 to 1.24.0 and TTS is still working for me, so that wasn't it. Still a good guess.

**Comment by sammyf** [comment]
I just updated to the latest text-generation-webui (always a risky business, "thanks" to Oobabooga's slightly unusual take about not using branches, avoiding testing and breaking stuff "because" ... ) and things still work :/ 

If @Juangea could tell me which module can not be found maybe I can figure out what's wrong on windows.

**Comment by bigsk1** [comment]
my guess is ffmpeg isn't installed and added to PATH

**Comment by sammyf** [comment]
yep. Would make sense. As a Linux user ffmpeg is basically essential for many tasks, so I didn't even think of that.

**Comment by bigsk1** [comment]
> yep. Would make sense. As a Linux user ffmpeg is basically essential for many tasks, so I didn't even think of that.

ya I just updated the comment, but ya there is a certain list of items you have to have messing around with oogabooga, Stable Diffusion, SillyTavern Koyha, and TTS projects. Cuda toolkit, Git, Python, Conda, Microsoft Visual Studio C++, node.js  and ffmpeg.

**Comment by rktvr** [comment]
after installing espeak, still get a numpy error

**Comment by sammyf** [comment]
can you run the TTS standalone? If not the issue is with with your installation of the package.

For details, check out :
https://github.com/mozilla/TTS#example-synthesizing-speech-on-terminal-using-the-released-models
